src/mono/wasm/templates/templates/console/app-support.mjs

```diff
@@ -166,7 +166,7 @@ try {
             if (runArgs.runtimeArgs.length > 0)
                 INTERNAL.mono_wasm_set_runtime_options(runArgs.runtimeArgs);
 
-            Object.assign(App, { MONO, BINDING, IMPORTS, Module, runArgs });
+            Object.assign(App, { MONO, INTERNAL, BINDING, IMPORTS, Module, runArgs });
```

Comment:
Why is this needed?

Reply:
It is needed here: https://github.com/dotnet/runtime/blob/main/src/mono/wasm/templates/templates/console/app-support.mjs#L49-L66

Without it, we don't call `mono_wasm_exit` and the exit code would be always 0.

Reply:
We should probably remove that `    if (App && App.INTERNAL) {` check anyway. It was useful in `test-main.js`, but not here, IIUC. Can we done in a follow up PR though.